pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -683,7 +683,7 @@ func (m *kubeGenericRuntimeManager) computePodActions(pod *v1.Pod, podStatus *ku
 func (m *kubeGenericRuntimeManager) SyncPod(pod *v1.Pod, podStatus *kubecontainer.PodStatus, pullSecrets []v1.Secret, backOff *flowcontrol.Backoff) (result kubecontainer.PodSyncResult) {
 	// Step 1: Compute sandbox and container changes.
 	podContainerChanges := m.computePodActions(pod, podStatus)
-	klog.V(3).InfoS("computePodActions got for pod", "podActions", podContainerChanges, "pod", klog.KObj(pod))
+	klog.V(3).InfoS("ComputePodActions got for pod", "podActions", podContainerChanges, "pod", klog.KObj(pod))
```

Comment:
same with [this](https://github.com/kubernetes/kubernetes/pull/100066#discussion_r591671060), `computePodActions` is method name.

Reply:
thank you for review. The code has updated.